src/pudl/metadata/resources/eia861.py

```diff
@@ -625,13 +633,13 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_reliability": {
-        "description": (
-            """Standardized metrics of electricity system reliability and outage
-impacts. Includes the system average interruption duration index (SAIDI), system average
+        "description": {
+            "additional_summary_text": "electricity system reliability and outage impacts",
```

Comment:
Sneaky missing period
```suggestion
            "additional_summary_text": "electricity system reliability and outage impacts.",
```